ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/Diffy.scala

```diff
@@ -36,10 +36,14 @@ object DeltaType extends Enumeration {
 sealed trait DeltaValue
 
 /** Delta value of unknown type. */
-case object UnknownDelta extends DeltaValue
+case object UnknownDelta extends DeltaValue {
+  override def toString: String = "UNKNOWN"
+}
 
 /** Delta value with a known type and computed difference. */
-case class TypedDelta(deltaType: DeltaType.Value, value: Double) extends DeltaValue
+case class TypedDelta(deltaType: DeltaType.Value, value: Double) extends DeltaValue {
+  override def toString: String = s"$deltaType\t$value}"
```

Comment:
did you mean to have a close bracket here on the end of the string

Reply:
Heh, no I did not

Reply:
Nice catch